architecture/additional_concepts/dynamic_admission_controllers.adoc

```diff
@@ -0,0 +1,540 @@
+[[architecture-additional-concepts-dynamic-admission-controllers]]
+= Dynamic Admission Controllers
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+
+== Overview
+
+Dynamic admission controllers allow you to deploy custom (out-of-tree)
+xref:../../architecture/additional_concepts/admission_controllers.adoc#architecture-additional-concepts-admission-controllers[admission control] plug-ins
+and configure them at runtime. 
+
+Admission control plug-ins intercept requests to the master API prior to persistence of a resource, but after the request is authenticated and authorized. The admission stage of the API server processing is one of the most powerful tools for securing an {product-title} cluster by restricting the objects that can be created. However, creating custom
```

Comment:
s/control plug-ins/controllers/. Can we use either "plug-in" or controllers? Both at the same time is confusing.